02-builder-analysis/exercises/zgz-bikes.md

```diff
@@ -0,0 +1,124 @@
+* *Features Highlighted*:
+  * Import: connect datasets from links
+  * Analysis: Create areas of influence, Intersect with second layer, Enrich with Data Observatory.
+  * Cartography: auto-style.
+* *Datasests needed*:
+  - Zaragoza blocks: [https://builder-demo.carto.com/api/v2/sql?q=SELECT+*+FROM+manzana&format=shp&filename=manzanas](https://builder-demo.carto.com/api/v2/sql?q=SELECT+*+FROM+manzana&format=shp&filename=manzanas), the original data source can be download it from [CartoCiudad](http://datos.gob.es/es/catalogo/e00125901-cartociudad).
+  - Public bikes anchors: [http://www.zaragoza.es/api/recurso/urbanismo-infraestructuras/estacion-bicicleta.geojson?rf=html&results_only=false&srsname=wgs84&rows=129](http://www.zaragoza.es/api/recurso/urbanismo-infraestructuras/estacion-bicicleta.geojson?rf=html&results_only=false&srsname=wgs84&rows=129), you can check what the API call returns copying [this url](http://www.zaragoza.es/api/recurso/urbanismo-infraestructuras/estacion-bicicleta?rf=html&results_only=false&srsname=wgs84&rows=129) into your browser.
+
+## Contents
+
+- [Create a map](#map)
+- [Analysis 1: Create areas of influence from bikes anchors](#aoi)
+- [Analysis 2: Intersect areas and city blocks](#intersect)
+- [Analysis 3: Assign total population to city blocks](#do)
+- [Widgets and auto-styling](#widgets)
+- [Publish](#publish)
+
+<hr>
+
+## Create a map <a name="map"></a>
+
+* Click on **`NEW MAP`**.
+* Click on **`CONNECT DATASET`**. Submit this url, choose any `Sync my data` option if sync table feature is available for your account, and make sure the `Let CARTO automatically guess data types and content on import` box is checked: [http://www.zaragoza.es/api/recurso/urbanismo-infraestructuras/estacion-bicicleta.geojson?rf=html&results_only=false&srsname=wgs84&rows=129](http://www.zaragoza.es/api/recurso/urbanismo-infraestructuras/estacion-bicicleta.geojson?rf=html&results_only=false&srsname=wgs84&rows=129). Click **`CONNECT DATASET`** button.
+* Click on **`CREATE MAP`**.
+* From the map's **`LAYERS`** pane click **`ADD`**. Click **`CONNECT DATASET`**. Connect the `Zaragoza blocks` dataset using the same steps as above, with this url: [https://builder-demo.carto.com/api/v2/sql?q=SELECT+*+FROM+manzana&format=shp&filename=manzanas)
+* Double-click on the map layer titles to rename them "Bike anchors" and "Blocks". You can also rename the map this way.
+
+![layers](imgs/zgzbikes/01-layers.png)
+
+
+## Create areas of influence from bikes anchors <a name="aoi"></a>
+
+* Click on "Bike anchors" layer.
+* Go to **`ANALYSIS`** tab.
+* Click on **`ADD ANALYSIS`**.
+* Select `Create areas of influence` analysis and click on **`ADD ANALYSIS`**.
+* Set `RADIUS` to `250` meters and `BOUNDARIES` to `Dissolved`, and click on **`APPLY`**.
+
+![aoi](imgs/zgzbikes/02-aoi.png)
+
+* In order to recover our points layer, go to the main menu and drag the layer source node out.
+
```

Comment:
Maybe break down this step a bit?

"...go back to the LAYERS pane. Click the original source of data {filename} from the _Bike anchors_ and drag it" above (or below..."